README.md

```diff
@@ -106,9 +109,9 @@ services:
 
```

Comment:
Should we add a section on how to Interact with the application using the frontend as well?